arch/xtensa/include/elf.h

```diff
@@ -34,4 +34,28 @@
 #define R_XTENSA_ASM_EXPAND 11
 #define R_XTENSA_SLOT0_OP   20
 
+#ifdef CONFIG_COREDUMP
```

Comment:
remove the check

Reply:
not address this suggestion yet. @sdc-g

Reply:
Hi @xiaoxiang781216
Sorry, let me confirm do you mean to remove #ifdef CONFIG_COREDUMP ?
But the CONFIG_COREDUMP also wrap the code which use EM_ARCH/EF_FLAG/elf_gregset_t

Reply:
but other arch define EM_ARCH without CONFIG_COREDUMP. And these macros and struct may be used in other context.

Reply:
Thanks for your comment. sorry for late reply.